verl/trainer/main_ppo.py

```diff
@@ -101,7 +101,13 @@ def run(self, config):
 
             actor_rollout_cls = AsyncActorRolloutRefWorker if config.actor_rollout_ref.rollout.mode == "async" else ActorRolloutRefWorker
             ray_worker_group_cls = NVMegatronRayWorkerGroup
+        elif config.actor_rollout_ref.actor.strategy == "spmd":
+            assert config.actor_rollout_ref.actor.strategy == config.critic.strategy
+            from verl.single_controller.ray import RayWorkerGroup
+            from verl.workers.tpu_workers import ActorRolloutRefWorker, AsyncActorRolloutRefWorker, CriticWorker
```

Comment:
We're not testing AsyncActorRolloutRefWorker, right? Is so, may be just remove the extra code related to it?